yarn.lock

```diff
@@ -409,6 +409,18 @@
   resolved "https://registry.yarnpkg.com/@humanwhocodes/object-schema/-/object-schema-2.0.1.tgz#e5211452df060fa8522b55c7b3c0c4d1981cb044"
   integrity sha512-dvuCeX5fC9dXgJn9t+X5atfmgQAzUOWqS1254Gh0m6i8wKd10ebXkfNKiRK+1GWi/yTvvLDHpoxLr0xxxeslWw==
 
+"@isaacs/cliui@^8.0.2":
```

Comment:
info Reasons this module exists
   - "tsup#sucrase#glob#jackspeak" depends on it
   - Hoisted from "tsup#sucrase#glob#jackspeak#@isaacs#cliui"